### ISSUE TITLE
Method answerWebAppQuery call request with type answerCallbackQuery

### ISSUE THREAD

**Issue by r1ddev** [yagop/node-telegram-bot-api#971]
> Bug Report

### Actual Behavior
`answerWebAppQuery` function not works propertly, telegram send error: "400 Bad Request: query is too old and response timeout expired or query ID is invalid".
In answerWebAppQuery function (telegram.js:1482) actually requested type `answerCallbackQuery`.
So the function `answerWebAppQuery` is not working.

### Expected Behavior
Function `answerWebAppQuery` must send type `answerWebAppQuery`.

### Example
Just replace it to `answerWebAppQuery` and all will works fine.

```JavaScript
answerWebAppQuery(webAppQueryId, result, form = {}) {
    form.web_app_query_id = webAppQueryId;
    form.result = stringify(result);
    return this._request('answerWebAppQuery', { form });
  }
```

**Comment by theexplay** [comment]
any updates? answerWebAppQuery api doesn't work cause of this problem

**Comment by starseedamm** [comment]
The answer to the problem described looks like it's right there above. 
**Problem**: "Function **answerWebAppQuery** must send type **answerWebAppQuery**."
**Solution**: "Just **replace** it to **answerWebAppQuery** and all will works fine."

**Comment by danielperez9430** [comment]
Fix in the new version v0.58.0
